src/server/admin-controller.js

```diff
@@ -118,6 +123,12 @@ class AdminController {
       needsPermission(permissions.ADMIN),
       (req, res) => this.handlePostCdnUploadDirectoryCreationRequest(req, res)
     );
+
+    router.get(
+      '/api/v1/admin/request-info',
+      needsPermission(permissions.ADMIN),
+      (req, res) => this.handleGetRequestInfoRequest(req, res)
+    );
```

Comment:
This new endpoint is there to find out the number of proxies the application is running behind on production (see also the  explanation in the ticket: https://educandu.atlassian.net/browse/EDU-976)